junit-jupiter-params/src/test/java/org/junit/jupiter/params/provider/MockCsvAnnotationBuilder.java

```diff
@@ -98,6 +108,24 @@ CsvSource build() {
 			when(annotation.emptyValue()).thenReturn(super.emptyValue);
 			when(annotation.nullValues()).thenReturn(super.nullValues);
 			when(annotation.maxCharsPerColumn()).thenReturn(super.maxCharsPerColumn);
+			when(annotation.ignoreTrailingAndLeadingWhitespace()).thenReturn(super.ignoreTrailingAndLeadingWhitespace);
+
+			// @CsvSource
+			when(annotation.value()).thenReturn(this.lines);
+
+			return annotation;
+		}
+
+		CsvSource build(boolean ignoreTrailingAndLeadingWhitespace) {
```

Comment:
This method is not necessary, as you can already set the `ignoreTrailingAndLeadingWhitespace` boolean with the `ignoreTrailingAndLeadingWhitespace()` method you introduced line 77.

Reply:
See https://github.com/junit-team/junit5/pull/2491#discussion_r545350282

Reply:
yes. but i need it for the "false" attribute value instead of using the setting in CsvSetting